src/services/CalculatorProcess.js

```diff
@@ -1,51 +1,43 @@
-let memory = [];
-let result = "";
-function calculatorProcess(content,visor) {
+let memory = "", process = "";
+let sLenth = 0;
+
+function CalculatorProcess(content, visor) {
+	const PLUS_MINUS_SIGNAL = "S";
+	const convertion = (string) => String(eval(string))
+	const firstInterations = ["S", "+", "-", "/", "%", "*", "="];
+	const clear = () => process = memory = "";
 	
-	const numbersOperators = () => String(eval((result)));
+	memory += content
 	
-	memory += content;
-
-	if (memory.includes("=")) {
-
-		result = memory.slice(0,-1);
-
-		if (memory.includes("+")) {
-			memory = numbersOperators();
+	firstInterations.forEach(interation => {
```

Comment:
I guess you don't nee to do a foreach, just check if firstInterations contains `content`

Reply:
ooh that's true, i thought that it couldn't run through the array without a loop, but it still works, quite interesting :p

Reply:
done